config/7.5.0/envs/rhdm-authoring-ha.yaml

```diff
@@ -474,7 +212,7 @@ others:
                         fieldRef:
                           apiVersion: v1
                           fieldPath: metadata.namespace
-                  image: "registry.redhat.io/amq-broker-7/[[$.Constants.BrokerImage]]:[[$.Constants.BrokerImageTag]]"
+                  image: "registry.redhat.io/amq7/[[$.Constants.BrokerImage]]:[[$.Constants.BrokerImageTag]]"
```

Comment:
@spolti : Don't we now have to add variable here for the first part of the image name?

"amq-broker-7" is fine for LastMinorVersion and LastMicroVersion here: https://github.com/kiegroup/kie-cloud-operator/blob/fab4d6a8f381bf3e2a21ea8ebd744b6aa3d45144/pkg/controller/kieapp/constants/constants.go#L73-L96 , however it is "amq7" in CurrentVersion. So, I think we'd have to come up with another variable, yes? Perhaps something like [[$.Constants.BrokerImageProduct]]?

If so, the line would be:
image: "registry.redhat.io/[[$.Constants.BrokerProduct]]/[[$.Constants.BrokerImage]]:[[$.Constants.BrokerImageTag]]"

@tchughesiv , wdyt?

Reply:
@errantepiphany its a thought but likely unnecessary as the config templates themselves are versioned, so 7.4.1 can have a different context setting in the templates themselves... e.g. -
https://github.com/kiegroup/kie-cloud-operator/blob/fab4d6a8f381bf3e2a21ea8ebd744b6aa3d45144/config/7.4.1/envs/rhdm-authoring-ha.yaml#L490

Reply:
@tchughesiv Ah, yeah, you're right. /facepalm. I totally missed that the config directory of @spolti 's change was itself "7.5.0". Nevermind!

Reply:
(Resolving this conversation as I stand corrected.)